app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -390,30 +390,38 @@ object AppRepository {
         metaDatabaseRepository.insert(values)
     }
 
+    fun readAlarmTimeIndex() =
```

Comment:
Why route everything through `AppRepository`? That'll make it even more of a god object. I'd make callers use `SettingsRepository` directly.

Why the `read` prefix? Why not `get`?

Reply:
The `AppRepository` hides whether data is persisted in shared preferences, a database or a file from the app layer above. The methods exposed by the `SharedPreferencesRepository` should only be accessed from the `AppRepository` layer.

I have chosen `read` following the CRUD schema.